dask-metrics/README.md

```diff
@@ -0,0 +1,132 @@
+# dask-metrics
+
+A tool for collecting metrics about the performance and utilization of GPUs on distributed Dask clusters.
+
+## Overview
+
+dask-metrics will collect information about how your cluster is performing in the background with little to no management necessary and minimal overhead.
+
+Existing workflows do not need to be modified to take advantage of dask-metrics. Simply connect to the cluster and start the monitor before submitting jobs and then stop the monitor when you're done.
+
+After metrics have been collected, the user can use dask-metrics to calculate statistics like peak memory usage or they can export those metrics to csv files to be processed and examined elsewhere.
+
+## Installation
+
+```bash
+conda install dask-metrics -c travishester
+```
+
+## Metrics Tracked
+
+### Always Tracked
+* **`job`**: the job that is currently executing.
+* **`dag`**: the dag within the job that is currently executing.
+* **`timestamp`**: the time, in seconds, since the start of metric collection.
+
+### Optional
+* **`total-mem`**: the memory used, in bytes, by each GPU on a worker
+* **`mem-util`**: the memory utilization of each GPU on a worker as a percentage of total memory available.
+* **`compute-util`**: the compute utilization of each GPU on a worker as a percentage of maximum compute ability.
+
+## How to Use
+
+Existing workflows to not need to be modified to use dask-metrics. The only code that needs to be added is at the beginning and end of your workflow.
+
+Before the jobs you want to monitor are submitted to the cluster, first attach a `Monitor` to a Client object, pass it a list of metrics to track, and start it.
+```python
+from dask_metrics import Monitor
+
+client = Client('[scheduler address]')  # Client connected to cluster
+monitor = Monitor(client)  # create Monitor object bound to client
+
+# attach monitor to cluster with list of metrics to track
+monitor.attach(['mem-util', 'compute-util'])
```

Comment:
Might be better to be explicit and add the argument name as well.
Eg: `monitor.attach(tracking=[...])`